Dockerfile

```diff
@@ -132,23 +130,40 @@ RUN mkdir -p /usr/src/humhub/protected/config/ && \
     rm -f var/www/localhost/htdocs/protected/config/common.php /usr/src/humhub/protected/config/common.php && \
     echo "v${HUMHUB_VERSION}" >  /usr/src/humhub/.version
 
+# Non-root directories
+RUN mkdir -p /var/www/localhost/htdocs/protected/runtime/logs && \
+    mkdir -p /var/www/localhost/htdocs/uploads /var/www/localhost/htdocs/assets /var/www/localhost/htdocs/protected/modules /var/www/localhost/htdocs/themes /var/www/localhost/htdocs/protected/config && \
+    mkdir -p /run/nginx /run/php-fpm && \
+    touch /var/www/localhost/htdocs/protected/runtime/logs/app.log && \
+    chown 100:101 -R /var/www/localhost/htdocs/protected/runtime/logs /run/nginx /run/php-fpm \
```

Comment:
for an openshift setup using the group 0 might be better than 101 because openshift gives the user a "random" high-id and the group 0. If we just allow the group 0 to read/write most parts of the files this should work in most scenarios.